engine-rest/engine-rest-openapi/src/main/templates/paths/external-task/{id}/errorDetails/get.ftl

```diff
@@ -0,0 +1,54 @@
+{
+
+  <@lib.endpointInfo
+      id = "getExternalTaskErrorDetails"
+      tag = "External Task"
+      desc = "Retrieves the error details in the context of a running external task by id." />
+
+  "parameters" : [
+
+    <@lib.parameter
+        name = "id"
+        location = "path"
+        type = "string"
+        required = true
+        last = true
+        desc = "The id of the external task for which the error details should be retrieved."/>
+
+  ],
+
+  "responses" : {
+
+    "200": {
+      "description": "Request successful.",
+      "content": {
+        "text/plain": {
+          "schema": {
+            "type": "string",
+            "description": "The error details for the external task."
+          },
+          "examples": {
+            "example-1": {
+              "value": "
+                org.apache.ibatis.jdbc.RuntimeSqlException: org.apache.ibatis.jdbc.RuntimeSqlException: test cause
+                  at org.camunda.bpm.engine.test.api.externaltask.ExternalTaskServiceTest.testHandleFailureWithErrorDetails(ExternalTaskServiceTest.java:1424)
+                  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
+                  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
+                  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
+                "
+            }
+          }
+        }
+      }
+    },
+
```

Comment:
```suggestion

    <@lib.response
        code = "204"
        desc = "Request successful. In case, the external task has no error details." />

```